tools/iwyu/cata.imp

```diff
@@ -1,4 +1,5 @@
 [
+    { ref: "cpp-17.stdlib.imp" },
```

Comment:
```suggestion
    { include: ["\"flexbuffer_json-inl.h\"", "private", "\"flexbuffer_json.h\"", "public"] },
    { ref: "cpp-17.stdlib.imp" },
```

Reply:
Currently `flexbuffer_json.h` does not have `#include flexbuffer_json-inl.h`, so this is not technically a reexport. Is this still correct? Are there any files that should explicitly include the -inl version?

(Thanks for the comment though, this specific part of the codebase is giving me a lot of trouble)

Reply:
It's at the bottom of the file.

Reply:
Not githubs fault, I edited the comment after.